libs/vkd3d/device.c

```diff
@@ -621,6 +621,10 @@ static const struct vkd3d_instance_application_meta application_override[] = {
      * Use no-staggered-submit by default on UE. We've only observed issues in Wukong here, but
      * unless we see proof that UE titles want staggered,
      * we'll disable for now to be defensive and de-risk any large scale regressions. */
+    /* Monster Hunter Wilds beta
+     * Game renders to a UAV | RTV image, but discards it, and then samples from it. Leading to black screen with FSR3 enabled.
+     * Native D3D12 AMD driver tends to disable compression for UAV in general. */
+    { VKD3D_STRING_COMPARE_STARTS_WITH, "MonsterHunterWilds", VKD3D_CONFIG_FLAG_DISABLE_UAV_COMPRESSION, 0 },
```

Comment:
Should probably be moved above the ue5 comment again, but yeah, game's doing *great* so far...

Reply:
Right ... I always frog that up.

Reply:
Fixed.